dio/lib/src/form_data.dart

```diff
@@ -48,7 +48,6 @@ class FormData {
     encodeMap(
       map,
       (key, value) {
-        if (value == null) return null;
```

Comment:
Allowing this will make null value converted to `"null"` eventually, which I think is unexpected for a valid `FormData`. It should be `""` instead.

Reply:
I agree. I will put:

```

if (value == null) {
          fields.add(MapEntry(key, ""));
        }
```

Reply:
Try L57 with `value?.toString() ?? ''`.

Reply:
Changes applied